platform/src/main/resources/anchor-config-defaults.yaml

```diff
@@ -250,10 +253,34 @@ data-spring-jdbc-sqlite:
   spring.datasource.username: admin
   spring.datasource.password: admin
   spring.mvc.converters.preferred-json-mapper: gson
+  spring.liquibase.enabled: false    # certain features that liquibase uses (ex: addForeignKeyConstraint) are not supported by sqlite
+
+data-spring-jdbc-aws-aurora-postgres:
+  spring.jpa.generate-ddl: true
+  spring.jpa.database: POSTGRESQL
+  spring.jpa.show-sql: false
+  spring.datasource.driver-class-name: org.postgresql.Driver
+  spring.datasource.url: jdbc:postgresql://database-2-instance-1.chizvyczscs2.us-east-1.rds.amazonaws.com:5432/
+  spring.datasource.username: postgres
+  spring.datasource.password: <PASSWORD>
+  spring.mvc.converters.preferred-json-mapper: gson
+  spring.liquibase.change-log: classpath:/db/changelog/db.changelog-master.yaml
+
+data-spring-jdbc-local-postgres:
+  spring.jpa.generate-ddl: true
+  spring.jpa.database: POSTGRESQL
+  spring.jpa.show-sql: false
+  spring.datasource.driver-class-name: org.postgresql.Driver
+  spring.datasource.url: jdbc:postgresql://localhost:5432/anchorplatform
+  spring.datasource.username: anchorplatform
+  spring.datasource.password: <PASSWORD>
+  spring.mvc.converters.preferred-json-mapper: gson
+  spring.liquibase.change-log: classpath:/db/changelog/db.changelog-master.yaml
```

Comment:
🤔 Is that ok to leave all of these database configurations uncommented? Just wondering if that could bring any side effects.

Reply:
I think we can leave them un-commented. The user may do cherry picking by enabling different database configurations.